index.html

```diff
@@ -0,0 +1,55 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="container">
+        <div class="wrapper">
+            <nav class="flex-nav box1">
+                <ul>
+                    <li>
+                        <h1 class="logo">My logo</h1>
+                    </li>
+                    <li class="logo-img">
+                        <i><img src="images/Union.svg" alt="logo"></i>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <div class="box2">
+            <div class="title">
+                <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
+            </div>
+
+            <div>
+                <p>I'm a software developer! I can help you build a product, feature or website 
+                    look through some of my work and experience! If you like what you see and have
+                    a project you need coded, don't hestiate to contact me.
+                 </p>
```

Comment:
- [ ] Great job 😃, but I see that you didn't change the color of the p tag, so kindly try to follow the color for it from the Figma Design 💯
- [ ] Also consider using the font weight of the p tag from the Figma design to let your portfolio look as a professional portfolio👌

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/212381010-1e762ff6-18c6-4dfa-937b-119c37f3ddbb.png)|    ![image](https://user-images.githubusercontent.com/83360636/212381068-7a71f418-a76f-433b-8388-3421f6a3fff2.png)|